.editorconfig

```diff
@@ -11,10 +11,9 @@ max_line_length = 80
 trim_trailing_whitespace = true
 
 [*.json]
-indent_style = tab
+indent_size = 4
```

Comment:
How come 4?

Reply:
Personally I prefer that in all languages. I can go with two if you prefer it?